spec/support/test_environment.rb

```diff
@@ -6,6 +6,7 @@
   config.include RspecTagMatchers
   config.include CustomMacros
   config.mock_with :rspec
+  config.deprecation_stream = '/dev/null'
```

Comment:
I'm not sure. We will drop rails 3 but this will not show deprecations for rails 4 etc.